cedar-policy/src/api.rs

```diff
@@ -1169,21 +1383,38 @@ pub struct EntityTypeName(ast::Name);
 
 impl EntityTypeName {
     /// Get the basename of the `EntityTypeName` (ie, with namespaces stripped).
+    /// ```
+    /// use cedar_policy::EntityTypeName;
+    /// use std::str::FromStr;
+    /// let type_name = EntityTypeName::from_str("MySpace::User").unwrap();
+    /// assert_eq!(type_name.basename(), "User");
+    /// ```
     pub fn basename(&self) -> &str {
         self.0.basename().as_ref()
     }
 
     /// Get the namespace of the `EntityTypeName`, as components
+    /// ```
+    /// use cedar_policy::EntityTypeName;
+    /// use std::str::FromStr;
```

Comment:
let's hide these imports
```suggestion
    /// # use cedar_policy::EntityTypeName;
    /// # use std::str::FromStr;
```